.yamllint

```diff
@@ -0,0 +1,46 @@
+---
+locale: en_US.UTF-8
+
+ignore:
+- ".tox"
+
+rules:
+  braces:
+    min-spaces-inside: 0
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
+  brackets:
+    forbid: non-empty
+  colons:
+    max-spaces-before: 0
+    max-spaces-after: 1
+  commas:
+    max-spaces-before: 0
+    min-spaces-after: 1
+    max-spaces-after: 1
+  comments:
+    require-starting-space: true
+    ignore-shebangs: true
+    min-spaces-from-content: 2
+  comments-indentation: {}
+  document-start:
+    present: true
+  empty-lines:
+    max: 1
+  empty-values:
+    forbid-in-block-mappings: true
+    forbid-in-flow-mappings: true
+    forbid-in-block-sequences: true
+  hyphens:
+    max-spaces-after: 1
+  indentation:
+    indent-sequences: false
+    spaces: 2
+  line-length: disable
+  key-duplicates: {}
+  new-line-at-end-of-file: {}
+  quoted-strings:
+    quote-type: double
```

Comment:
double quote allows escape and single quote just literal, should we use `any` to avoid `# yamllint disable-line rule:quoted-strings`?